pkg/mcp/mcp.go

```diff
@@ -2,362 +2,164 @@ package mcp
 
 import (
 	"context"
+	"errors"
+	"os/exec"
+	"strings"
 
-	"github.com/mark3labs/mcp-go/mcp"
-	"github.com/mark3labs/mcp-go/server"
+	"github.com/modelcontextprotocol/go-sdk/mcp"
 )
 
-var TEMPLATE_RESOURCE_URIS = []string{
-	"https://github.com/functions-dev/templates",
-}
+const (
+	Name    = "func"
+	Title   = "func"
+	Version = "0.1.0"
```

Comment:
Encapsulate non exported.
```suggestion
	bame    = "func"
	title   = "func"
	version = "0.1.0"
```